Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -94,7 +95,7 @@ public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, Defi
         if (!Resolve(uid, ref component))
             return false;
 
-        if (!_toggle.IsActivated(uid))
+        if (!_toggle.IsActivated(uid) && !component.IgnoreToggle) // Imp - Multitool defib
```

Comment:
multitools dont have ItemToggle so itll always be treated as activated anyway, not needed